eng/Directory.Build.Common.targets

```diff
@@ -167,6 +167,14 @@
     </NoWarn>
   </PropertyGroup>
 
+  <!-- Suppress warnings from duplicated names analyzer for client libraries -->
+  <PropertyGroup Condition="'$(IsClientLibrary)' == 'true'">
```

Comment:
don't exclude the analyzer for all client libraries. Exclude individual violations explicitly in each project